src/components/reviews.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from "react";
+import { fetchReviews } from "../api";
+
+const Reviews = () => {
+    const [reviewsData, setReviews] = useState([]);
+
```

Comment:
Another state needs to be added for letting the user know if the reviews are still loading.